x-pack/plugins/data_enhanced/public/plugin.ts

```diff
@@ -52,6 +55,14 @@ export class DataEnhancedPlugin
 
   public start(core: CoreStart, plugins: DataEnhancedStartDependencies) {
     setAutocompleteService(plugins.data.autocomplete);
+
+    core.chrome.setBreadcrumbsAppendExtension({
+      content: toMountPoint(
+        React.createElement(
+          createConnectedBackgroundSessionIndicator({ sessionService: plugins.data.search.session })
```

Comment:
This new api is used from within `data_enhanced` plugin. 
There is no need (at least for now) for apps to control this indicator on their own as source of truth of session state is data plugin

Reply:
(Copy of my reply on slack)

```ts
export type ChromeBreadcrumb = EuiBreadcrumb;
```

```ts
export type EuiBreadcrumb = CommonProps & {
	    /**
	     * Visible label of the breadcrumb
	     */
	    text: ReactNode;
```

Why not just use a reactNode with your 'extension' component when setting the breadcrumb (instead of plain text)?

Reply:
(Copy my reply on slack 😅)

How I looked at it until now:
I don’t want apps to add this to their breadcrumbs.
instead I want data plugin to add this to a breadcrumb. This indicator will be rendered when a page is using background sessions